src/Lumi/Styles/Box.purs

```diff
@@ -61,6 +66,9 @@ _align a = styleModifier_ $ css { alignItems: prop a }
 _alignSelf :: forall props. FlexAlign -> PropsModifier props
 _alignSelf a = styleModifier_ $ css { alignSelf: prop a }
 
+_fill :: forall props. PropsModifier props
+_fill = styleModifier_ $ css { flex: str "1" }
```

Comment:
I'm not sure about this one.. it doesn't behave as expected depending on the flex direction, parent styles, and sibling styles. I'd rather call it `_flex`.

Reply:
Hm, yes, good point! I'll fix that.